.github/workflows/poetry.yml

```diff
@@ -55,7 +55,8 @@ jobs:
             python-version: '3.10'
             toxenv-factor: 'locked'
           - # test with the lowest dependencies
-            python-version: '3.6'
+            os: ubuntu-20.04
```

Comment:
:x: remove the `os` - it uis unused in this matrix

Reply:
done!